libs/hmpb/render-backend/src/generate_fixtures.ts

```diff
@@ -15,21 +15,29 @@ import {
   generalElectionFixtures,
 } from './ballot_fixtures';
 
-function generateBallotFixture(
+async function generateBallotFixture(
   fixtureDir: string,
   label: string,
   document: Document
-): void {
+) {
+  // eslint-disable-next-line no-console
+  console.log(
+    `Generating: ${join(fixtureDir.replace(fixturesDir, ''), label)}`
+  );
   fs.writeFileSync(
     join(fixtureDir, `${label}-document.json`),
     JSON.stringify(document, null, 2)
   );
   const pdf = renderDocumentToPdf(document);
-  pdf.pipe(fs.createWriteStream(join(fixtureDir, `${label}.pdf`)));
+  const fileStream = fs.createWriteStream(join(fixtureDir, `${label}.pdf`));
+  pdf.pipe(fileStream);
```

Comment:
Can you use [`pipeline` from `streams/promises`](https://nodejs.org/api/stream.html#streampipelinestreams-options)?

Reply:
It's a little weird in this case because you have to call `pdf.end()`. But I found `finished` in `streams/promises` which is an improvement over what I have